docs/gitbook/tutorials/prometheus-operator.md

```diff
@@ -0,0 +1,144 @@
+# Flagger with Prometheus Operator
+
+This guide will show you how to use Flagger and Prometheus Operator
+This guide will handle only Blue/Green Deployment
+
+## Prerequisites
+
+Flagger requires a Kubernetes cluster **v1.11** or newer and NGINX ingress **0.24** or newer.
+
+Install Prometheus-Operator with Helm v3:
+
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com
+helm repo update
+kubectl create ns prometheus
+helm upgrade -i prometheus stable/prometheus-operator \
+--namespace prometheus \
+--set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValue=false
+```
+The `prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValue` allows Prometheus-Operator to watch serviceMonitor outside of his namespace.
+
+Install Flagger with Helm v3:
+
+```bash
+helm repo add flagger https://flagger.app
+helm repo update
+kubectl create ns flagger
+helm upgrade -i flagger flagger/flagger \
+--namespace flagger
+--set prometheus-prometheus-oper-prometheus.prometheus:9090 \
```

Comment:
I just came accross this PR, which is very useful to me. So if you guys don't mind me adding comment.

This line looks weird to me, are you trying to set `metricsServer` ?